pupa/importers/events.py

```diff
@@ -55,4 +67,33 @@ def prepare_for_db(self, data):
         data['start_time'] = gdt(data['start_time'])
         data['end_time'] = gdt(data.get('end_time', None))
 
+        for participant in data['participants'] :
+            if 'person_id' in participant:
+                participant['person_id'] = self.person_importer.resolve_json_id(
+                    participant['person_id'], 
+                    allow_no_match=True)
+            elif 'organization_id' in participant:
+                participant['organization_id'] = self.org_importer.resolve_json_id(
+                    participant['organization_id'], 
+                    allow_no_match=True)
+
+        for item in data['agenda'] :
+            for entity in item['related_entities'] :
+                if 'person_id' in entity :
```

Comment:
Can you remove the spaces before the final colons in if statements (multiple cases)?